sources/packages/web/src/components/aest/students/ApplicationHeaderTitle.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <detail-header :headerMap="headerMap" />
```

Comment:
Minor. Recently we follow this format for the Vue component attribute names. 

![image](https://github.com/user-attachments/assets/dc51819d-659e-4dd4-a447-7a8862910bf3)

Reply:
updated